GrpcClient/EmployeeGrpcRepository.cs

```diff
@@ -0,0 +1,286 @@
+﻿using Grpc.Core;
+using Grpc.Net.Client;
+using RepositoryContracts;
+
+namespace GrpcClient;
+
+public class EmployeeGrpcRepository : IEmployeeRepository
+{
+    private string _grpcAddress { get; set; }
+
+    public EmployeeGrpcRepository()
+    {
+        _grpcAddress = "http://192.168.140.143:50051";
+    }
+
+    public async Task<Entities.Employee> AddAsync(Entities.Employee employee)
+    {
+        using var channel = GrpcChannel.ForAddress(_grpcAddress); 
+        //create client for employee entity, "EmployeeClient" method is auto-genereted               
+        var client = new Employee.EmployeeClient(channel);
+        //reply - "AddSingleEmployee(newEmployeeDto)" comes from .proto file + need to convert the og-DTO to .proto-DTO
```

Comment:
what is og-DTO and has it been transformed to .proto-DTO?

Reply:
og dto means the one you can find in cs project and proto dto means the one initialized in proto file and it is used as a standart for comm. with java

Reply:
and where exactly is this one specific dto and what data it takes?